### DISCUSSION TITLE
Network Files: Open a modal feature

### DISCUSSION THREAD

**Discussion by ITFO** [microsoft-search/pnp-modern-search#3023] [Ideas]
**Is your feature request related to a problem? Please describe.**
Currently I include our Network drive as a source in our search. I was able to use it with the pnp search but since this file is hosted in a network drive I want user to open it directly on their browser

**Describe the solution you'd like**
Currently in the OOB search, when you add you network drive in your search vertical when you open a file sharepoint will open a modal for you to allow you to copy the network address of the file or maybe allows them to open network files directly from their browser

![1](https://user-images.githubusercontent.com/37436984/118576535-9247d000-b73d-11eb-819a-7775c66ffbf2.PNG)

![2](https://user-images.githubusercontent.com/37436984/118576542-95db5700-b73d-11eb-89ab-88f116b3c285.PNG)

**Comment by wobba** [comment]
This should ideally be a custom web component to solve the non https schemas which browsers block.
